src/badfish/main.py

```diff
@@ -109,7 +112,7 @@ async def error_handler(self, _response, message=None):
             raise BadfishException(detail_message)
 
     @alru_cache(maxsize=64)
-    async def get_request(self, uri, _continue=False, _get_token=False):
+    async def get_request(self, uri, _continue=False, _get_token=False, no_cache=None):
```

Comment:
lol, nice hack!

Reply:
sure, that's a probably nicer solution and not having to use random is a definite plus, will try to fix it 😅